app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx

```diff
@@ -142,6 +142,20 @@ class DiscoveryMap extends React.Component {
           </div>
         </div>
       );
+    } else {
+      const idsField = currentTab === "samples" ? "sample_ids" : "project_ids";
+      const count = sumBy(p => size(p[idsField]), values(mapLocationData));
+      return (
+        <div className={cs.bannerContainer}>
```

Comment:
Can you make it a component? 
Just think components whenever it is not a custom layout placement :)